Library/Homebrew/cask/audit.rb

```diff
@@ -550,6 +554,33 @@ def check_download
       add_error "download not possible: #{e}"
     end
 
+    def check_signing
+      return if !signing? || download.blank? || cask.url.blank?
+
+      odebug "Auditing signing"
+      odebug cask.artifacts
+      artifacts = cask.artifacts.select { |k| k.is_a?(Artifact::Pkg) || k.is_a?(Artifact::App) }
+
+      return if artifacts.empty?
+
+      downloaded_path = download.fetch
+      primary_container = UnpackStrategy.detect(downloaded_path, type: @cask.container&.type, merge_xattrs: true)
+
+      return if primary_container.nil?
+
+      Dir.mktmpdir do |tmpdir|
+        tmpdir = Pathname(tmpdir)
+        primary_container.extract_nestedly(to: tmpdir, basename: downloaded_path.basename, verbose: false)
+        cask.artifacts.each do |artifact|
```

Comment:
I think this is the problem. We want `artifacts` not `cask.artifacts`, right?

Reply:
I think you're right here 😅